### ISSUE TITLE
jQueryPP.com audit

### ISSUE THREAD

**Issue by chasenlehara** [bitovi/jquerypp#128]
Go through the website and look for broken demos & links, missing images & videos, spelling & grammar issues, accessibility issues, poor performance, etc.
